seglearn/transform.py

```diff
@@ -629,6 +630,169 @@ def transform(self, X, y=None, sample_weight=None):
         return Xt, yt, swt
 
 
+class StackedInterp(BaseEstimator, XyTransformerMixin):
+    '''
+    Extension of Interp where input is in stacked format.
+    Transformer for resampling time series data to a fixed period over closed interval
```

Comment:
Here you should clearly describe the 'stacked' format. 
Specifically indicate it must be at least 3 columns (time, var_type, var_value)
Additional columns are treated as additional channels for the variable eg (time, var_type, var_value1, var_value2)

Reply:
also indicate that each time series must include all the variables